code/modules/mod/mod_theme.dm

```diff
@@ -317,7 +317,7 @@
 	resistance_flags = FIRE_PROOF|LAVA_PROOF
 	max_heat_protection_temperature = FIRE_SUIT_MAX_TEMP_PROTECT
 	min_cold_protection_temperature = FIRE_SUIT_MIN_TEMP_PROTECT
-	complexity_max = DEFAULT_MAX_COMPLEXITY - 5
+	complexity_max = DEFAULT_MAX_COMPLEXITY
```

Comment:
you already lower the module value of a bunch of the modules this uses (and useful sidegrades for it like the thermal regulator). you aren't meant to be able to put everything into every suit.

Reply:
The only thing I lowered the value of was the drill module. In total, it still costs 9 complexity. My arguments still hold. 

The mining modsuit is underused because everything it has is a sidegrade of pre-existing mining gear, while locking you into it and making you unable to get many of the lavaland items. 

Making it utterly restricted in any modules to actually upgrade the frickin' thing just means there is NO reason to waste 2k credits on what amounts to worse drake armor, worse plasma cutter, awkward bag of holding, and a drastic reduction in storage size. It's useless by itself. 

Letting it actually GET valuable upgrades without cannibalizing it to the point you have no reason to use it gives it one small reason to be picked over normal lavaland equipment.

'You arent meant to put everything into one suit' doesn't hold because the everything that's in the suit is completely redundant, and by removing those you discard the reasons to purchase the thing to begin with.

Reply:
I think at this point I have to agree that the mining modsuit doesn't quite feel like a value purchase unless topside has completely stagnated in research for some reason, which is absolutely worst case scenario. While it technically does provide extra storage in that it saves a few pieces of equipment in pockets/your bag (namely the GPS, ore satchel and pickaxe/shovel), as well as carrying objects in crates (but not bodies obviously), you can't upgrade it with other modules at all. That limitation is especially harsh given the poor EVA capabilities.

I absolutely agree that since I have to cannibalize the suit to give it alternative functionality for some reason or another, I would ultimately be better off using a different modsuit from robotics and sticking with my standard gear and suits. It's only main draw at that point is the potential for bigger bag space, something capable even with a standard modsuit,  which also gives me inbuilt EVA. And by doing so, I can fit the modsuit with more modules overall, becoming more flexible as a result. A very mild slowdown from a standard suit hardly matters.

The mining bomb is neat but gimmicky, and majorly slower than equivalent mining gear at the point you can buy the suit. I did like the lava traversal, but not enough for me to stick to the suit. Everything else but the ash armor is replicable with any modsuit.